composer.json

```diff
@@ -19,7 +19,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^5.3.9|~7.0",
-        "symfony/framework-bundle": "^2.3.27|^2.6.6|~3.0",
```

Comment:
This didn't make sense at all. Because `^2.6.6` will never trigger as ^2.3.27 also passes 2.6.0 for example.